src/platform/forms-system/src/js/containers/FormPage.jsx

```diff
@@ -18,22 +18,10 @@ function focusForm() {
   focusElement('.nav-header > h2');
 }
 
-const scroller = Scroll.scroller;
-const scrollToTop = () => {
-  scroller.scrollTo(
-    'topScrollElement',
-    window.Forms?.scroll || {
-      duration: 500,
-      delay: 0,
-      smooth: true,
-    },
-  );
-};
-
 class FormPage extends React.Component {
   componentDidMount() {
     if (!this.props.blockScrollOnMount) {
-      scrollToTop();
+      scrollToTop('topScrollElement', window.Forms?.scroll || undefined);
```

Comment:
Same thing here: https://github.com/department-of-veterans-affairs/vets-website/pull/18770#discussion_r712523738

Reply:
> I'm going to hold making any adjustments to these. There was another request for additional refinement having to do with these and once that's merged, this will be handled by the `getScrollOptions` function. Any inline declarations like this will be replaced with that function so this will be a non-issue. 

Quoting my reply from another thread - putting this on hold til the other PR is merged that centralizes the `getScrollOptions` function and then will update this and test accordingly.